src/events/components/ImageView/EventColumn.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { IEventColumn } from 'events/models/Event';
+
+import style from './image.less';
+import LargeEvent from './LargeEvent';
+import LargeEventPlaceholder from './LargeEventPlaceholder';
+import SmallEventColumn from './SmallEvent';
+
+const EventColumn = ({ events, event_type }: IEventColumn) => {
+  return (
+    <div className={style.eventColumn}>
+      {events[0] ? (
+        <>
+          <LargeEvent {...events[0]} />
+          <SmallEventColumn events={events.slice(1, 4)} />
+        </>
+      ) : (
+        <LargeEventPlaceholder event_type={event_type} />
```

Comment:
Also wondered if it was possible to make this child component inherit event_type from its parent component or not

Reply:
It would be possible with object `rest` and `spread`, but I don't think it makes sense here.
I think this is the way to go :smile: